config/README.md

```diff
@@ -0,0 +1,100 @@
+Firebase Remote Config Quickstart
+==============================
+
+The Firebase Remote Config Web quickstart app demonstrates using Remote
+Config to define user-facing text in a web app.
+
+Introduction
+------------
+
+This is a simple example of using Remote Config to override in-app default
+values by defining service-side parameter values in the Firebase console. This
+example demonstrates a small subset of the capilities of Firebase Remote
+Config. To learn more about how you can use Firebase Remote Config in your app,
+see
```

Comment:
I feel like we need "the" (as in "the RC introduction").

Reply:
Not sure what you are suggesting here, could you be more specific?